sdg_mapping/sdg_index/sdg_index_utils.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import pandas as pd
+
+import sys
+import os
+import json
+
+from sdg_mapping import project_dir
+
+with open(f'{project_dir}/data/aux/sdg_index_data_opts.json', 'r') as f:
+    state_dict = json.load(f)
+
+
+def sdg_index_file_path(year):
+    """sdg_index_file_path
+    To fetch filepath of SDG Index datasets.

```

Comment:
```suggestion
    Generates file path for SDG Index datasets.

```

In this project we're using "fetch" as a term for downloading data from the web, so best not to confuse things.